NAMESPACE

```diff
@@ -16,6 +16,7 @@ export("coeftostring")
 export("createTexreg")
 exportMethods("extract")
 S3method("print", "texregTable")
+export("extract.broom")
```

Comment:
Can you please ensure that the `extract` methods are listed in the correct alphabetical order?